21.1 mz dashboards eud (#135)
DevExpress/dotnet-eud: dashboard-for-web/articles/web-dashboard-designer-mode/ui-elements/dialogs-and-wizards/dashboard-data-source-wizard.md

---
title: Dashboard Data Source Wizard
author: Margarita Zakhodyaeva
---
# Dashboard Data Source Wizard

The **Dashboard Data Source Wizard** allows you to create a new data source or edit the existing data sources.

> To invoke this wizard, go to the **Data Sources** page of the [dashboard menu](../dashboard-menu.md) and click **Add** button.

Reviewer: ```suggestion
To invoke this wizard, go to the **Data Sources** page of the [dashboard menu](../dashboard-menu.md) and click **Add**.
```

![](../../../../images/dashboard-data-sources-add-button.png)

> In the invoked window choose the existing data sources or click **Create data source**....

Reviewer: ```suggestion
In the invoked window, choose the existing data source or click **Create data source**....
```

![](../../../../images/dashboard-create-data-source.png) 

On the start page, select the data source type. You can create a new dashboard data source based on a database, JSON data, or an OLAP cube.

* [Database](dashboard-data-source-wizard/specify-data-source-settings-database.md)

>   Obtains data from all major data providers (Microsoft SQL Server, XML data, Microsoft Access, Oracle, etc.).

Reviewer: ```suggestion
  Obtains data from all major data providers (Microsoft SQL Server, XML data, Microsoft Access, Oracle, and so forth).
```
